pset-iterators.js

```diff
@@ -93,6 +105,13 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const presidentialParties = presidents.map((president) => {
```

Comment:
@kcycle nice use of `.map()` to build a new array that consists solely of president names